app/routes/individual_response.py

```diff
@@ -250,4 +254,7 @@ def individual_response_text_message_confirmation(schema, questionnaire_store):
     return render_template(
         template="individual_response/confirmation-text-message",
         mobile_number=mobile_number,
+        page_title=individual_response_handler.get_full_page_title(
```

Comment:
Not sure get_full_page_title adds much for me, in the other handlers we do similar, but we don't call it full_page_title

Reply:
are you suggesting just calling it `page_title`?

Reply:
yep

Reply:
Done in 864439b9